openapi/tags/CompactIdentifierBasedURIs.md

```diff
@@ -8,7 +8,7 @@ See the documentation on the [n2t.net](https://n2t.net/e/compact_ids.html) and [
 
 ## Calling Meta-Resolver APIs for Compact Identifier-Based DRS URIs
 
-Clients resolving Compact Identifier-based URIs need to convert a prefix (e.g. “drs.42”) into an URL pattern. They can do so by calling either the identifiers.org or the n2t.net API, since the two meta-resolvers keep their mapping databases in sync.
```

Comment:
I personally prefer "an", since I read it to myself as "an earl", not as "a you are ell". But the Web agrees with you (32M vs. 1M).